src/components/NavigationTabsComponent.jsx

```diff
@@ -0,0 +1,236 @@
+import Box from '@mui/material/Box'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import { isFunction, isNil } from 'lodash/fp.js'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import SignInButton from 'src/components/SignInButton.js'
+
+
+const styles = {
+  mainTab: {
+    padding: '0 25px',
+    fontSize: '16px',
+    textTransform: 'none',
+    fontFamily: 'Montserrat, sans-serif',
+    minHeight: '80px',
+    color: 'rgba(255, 255, 255, 0.7)',
+  },
+  mainTabActive: {
+    padding: '0 25px',
+    fontSize: '16px',
+    fontFamily: '\'Montserrat\', sans-serif',
+    minHeight: '80px',
+    textTransform: 'none',
+    color: 'white',
+    background: 'rgba(255, 255, 255, 0.1)',
+    fontWeight: 'bold'
+  },
+  subTab: {
+    padding: '0 25px',
+    fontSize: '15px',
+    textTransform: 'none',
+    fontFamily: '\'Montserrat\', sans-serif',
+    color: '#00609f',
+    minHeight: '65px'
+  },
+  subTabActive: {
+    padding: '0 25px',
+    fontSize: '15px',
+    textTransform: 'none',
+    fontFamily: '\'Montserrat\', sans-serif',
```

Comment:
Per line 15, Montserrat doesn't need quoting, so we can denoise:

```suggestion
    fontFamily: 'Montserrat, sans-serif',
    minHeight: '80px',
    textTransform: 'none',
    color: 'white',
    background: 'rgba(255, 255, 255, 0.1)',
    fontWeight: 'bold'
  },
  subTab: {
    padding: '0 25px',
    fontSize: '15px',
    textTransform: 'none',
    fontFamily: 'Montserrat, sans-serif',
    color: '#00609f',
    minHeight: '65px'
  },
  subTabActive: {
    padding: '0 25px',
    fontSize: '15px',
    textTransform: 'none',
    fontFamily: 'Montserrat, sans-serif',
```